examples/nextjs-with-typescript/pages/MuxPlayer.tsx

```diff
@@ -329,6 +330,7 @@ function MuxPlayerPage({ location }: Props) {
           defaultDuration={state.defaultDuration}
           playbackRate={state.playbackRate}
           playbackRates={state.playbackRates}
+          proudlyDisplayMuxBadge={state.proudlyDisplayMuxBadge}
```

Comment:
nitpick(non-blocking): consider adding a `<BooleanRenderer/>` for the attribute in our config form for easier testing